tests/playbook.yml

```diff
@@ -9,5 +9,4 @@
       with_items:
         - net-tools
   roles:
-    - role: java
```

Comment:
Deleting this line will make Vagrant tests fail. I wouldn't delete it but, in case of deletion, `requirements.yml` makes no sense.